administrator/components/com_patchtester/script.php

```diff
@@ -48,6 +51,24 @@ public function __construct()
 	 */
 	public function postflight($type, $parent)
 	{
-		$this->removeFiles();
+		if ($type == 'install')
+		{
+			$language = JFactory::getLanguage();
+			$language->load('com_patchtester', JPATH_ADMINISTRATOR, null, true);
+			$language->load('com_patchtester', JPATH_SITE, null, true);
+			echo '<h1><img src="../media/com_patchtester/images/icon-48-patchtester.png"> ' . Text::_('COM_PATCHTESTER') . '</h1>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_XML_DESCRIPTION') . '</p>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_patchtester">' . Text::_('COM_PATCHTESTER') . '</a></p>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_config&view=component&component=com_patchtester#authentication	&path=&return=aHR0cDovL2xvY2FsaG9zdC9wYXRjaDRhMTIvYWRtaW5pc3RyYXRvci9pbmRleC5waHA%2Fb3B0aW9uPWNvbV9wYXRjaHRlc3Rlcg%3D%3D">' . Text::_('COM_PATCHTESTER_OPTIONS') . '</a></p>';
```

Comment:
We need to make this one language string because the order of the texts can be different in other languages. So we will need something like this:
```php
Text::_('COM_PATCHTESTER_GOTO_PATCHTESTER');
```
In the language file we have:
```ini
COM_PATCHTESTER_GOTO_PATCHTESTER="Go to <a href=""index.php?option=com_patchtester"" alt="Go to Patch Tester">Patch Tester</a>" 
```

The returnurl doesn't decode properly for me and it contains your hostname, so i want to suggest to use this one. Which is /administrator/index.php?option=com_patchtester
```
/administrator/index.php?option=com_config&view=component&component=com_patchtester&path=&return=L2FkbWluaXN0cmF0b3IvaW5kZXgucGhwP29wdGlvbj1jb21fcGF0Y2h0ZXN0ZXI=
```
The string we can use then is:
```
Text::_('COM_PATCHTESTER_GOTO_PATCHTESTER_OPTIONS');
```
Like the other one write the full text with the anchor link.

Reply:
So if I update as suggested above that will be alright?

Reply:
Correct